.github/workflows/Extension_test.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Extension Test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+        paths:
+          - "puppet/**"
```

Comment:
Why are you restricting the path to this ?

Reply:
trigger the CI/CD pipeline when changes are made. In this case, it is set to "puppet/**".

Reply:
That seems incorrect. Shouldn’t it be earth

Reply:
i tried using /earth then it will not find the path for spec file and return error .that's way i specific the base repo path and then it execute the test write in folder using jasmine .